ngraph/core/src/op/convert.cpp

```diff
@@ -117,7 +119,7 @@ namespace convert
 bool op::v0::Convert::evaluate(const HostTensorVector& output_values,
                                const HostTensorVector& input_values) const
 {
-    NGRAPH_OP_SCOPE(v0_Convert_evaluate,
-                    return convert::evaluate_convert(input_values[0], output_values[0]));
+    NGRAPH_OP_SCOPE(v0_Convert_evaluate)
+    return convert::evaluate_convert(input_values[0], output_values[0]);
```

Comment:
It is better use `{}` always. Otherwise someone can add one more line before `return` and broke CC.

Reply:
It can work as `if` construction.
C++ allow to have two construction of if:
```
// inline
if (...)
    ...;
// And with {}
if (...) {
    ...
}
```

I used both constructions in order to check that both approaches work well.

Reply:
@ilyachur I see. I mean that other developers can be not aware about CC specific and can add something also.

Reply:
Ok, right. I got your point.  Fixed